python/spglib/spglib.py

```diff
@@ -73,12 +73,49 @@ class SpglibError(object):
 def get_version():
     """Return version number of spglib with tuple of three numbers.
 
+    Deprecated since v2.3.0
```

Comment:
Can you update ChangeLog.md to inform the deprecation?
CC: @atztogo

Reply:
Can you check the message there and where else it should be added? Probably the python api doc and `release.md`?

Reply:
The changelog messages look nice. Yes, can you add the deprecation message also in the python api?